### DISCUSSION TITLE
How can I mark a pytest_plugin to be autoused?

### DISCUSSION THREAD

**Discussion by mbkroese** [pytest-dev/pytest#9098] [Q&A] [answered]
I have a package `test_fixtures.submodule` that contains a fixture `database`.
In the tests of _another_ package (let's call it `foo`) I want to autouse this fixture.
How can I mark the `database` fixture defined in another package for auto usage?

**Comment by RonnyPfannschmidt** [comment, marked as answer]
Create a empty autouse fixture called like _require_database(database) in the belonging conftests

(sent from mobile)

**Reply by mbkroese** [reply]
Makes sense, thanks!
